src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -147,13 +147,33 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     
     String cleanedAndEmojiMessage = EmojiParser.parseToUnicode(cleanedMessageContent);
 
+    String mentionedUser = "";
+
+    int end = 0;
+
+    for (int i = 0; i < cleanedAndEmojiMessage.length(); i++){
+      if ((Character.toString(cleanedAndEmojiMessage.charAt(i))).equals("@")){
+        for (int j = i; j < cleanedAndEmojiMessage.length(); j++){
+          if ((Character.toString(cleanedAndEmojiMessage.charAt(j))).equals(" ")){
+            end = j; 
+          }
+
+        }
+      
+      mentionedUser = cleanedAndEmojiMessage.substring(i,end+i); 
```

Comment:
I think you should use regex instead.

Create a regex pattern along the lines of "@.+\s" which is equivalent to @, one or more characters, and a space, and then search for all matches. That'll give you the list of mentions